### ISSUE TITLE
training curve of CLIP

### ISSUE THREAD

**Issue by bladewaltz1** [openai/CLIP#266]
Hi, thanks for the great work!

I am trying to train a CLIP model on 1M image-text pairs with a batch size of 1920, and the training loss converges to around 3 after 50 epochs. I am wondering if this is a normal behavior? I would be really appreciated if you can provide me the training loss curve of the original CLIP model, or at least let me know the final loss value of the CLIP model.

Thank you very much!

**Comment by krishanrana** [comment]
I am having a similar issue and not sure if i should keep letting it train for longer as it there seems to still be a slow downward trend?
